pkg/descheduler/strategy_migration.go

```diff
@@ -38,123 +39,98 @@ import (
 // without any wiring. Keeping the wiring here so the descheduler can still use
 // the v1alpha1 configuration during the strategy migration to plugins.
 
-var pluginsMap = map[string]func(ctx context.Context, nodes []*v1.Node, params *api.StrategyParameters, handle *handleImpl){
-	"RemovePodsViolatingNodeTaints": func(ctx context.Context, nodes []*v1.Node, params *api.StrategyParameters, handle *handleImpl) {
+var strategyParamsToPluginArgs = map[string]func(params *v1alpha1.StrategyParameters) (*api.PluginConfig, error){
```

Comment:
I don't think it's necessary given the map will be dropped eventually. Though, if you believe there's a value in it, by all means :)

Reply:
it's temporary, but it is holding things together for a few releases. So testing this would be good to make sure we don't break anything in transition